### PR TITLE
Console - API properties screen migration 

### PR DESCRIPTION
**Pull request by ThibaudAV**
## Issue

https://gravitee.atlassian.net/browse/APIM-2363

## Description

Start to migrate API properties screen 

image

## Additional context

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-xeashgkmlx.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/src/management/api/proxy/properties-ng/api-properties-old.component.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export class ApiPropertiesComponent implements OnInit, OnDestroy {
 
   constructor(
     private readonly apiService: ApiV2Service,
-    private readonly apiPropertiesService: ApiPropertiesService,
+    private readonly apiPropertiesService: ApiPropertiesOldService,
```
Comment (jansobczynski) [inline gravitee-apim-console-webui/src/management/api/proxy/properties-ng/api-properties-old.component.ts:56]: Do we rename `private readonly apiPropertiesService: ApiPropertiesOldService,` to `private readonly apiPropertiesOldService: ApiPropertiesOldService,` too?

**Review by jansobczynski — APPROVED** [review]

File: gravitee-apim-console-webui/src/management/api/proxy/properties-ng/api-properties-old.component.ts (hunk 1 above)
Comment (phiz71) [reply gravitee-apim-console-webui/src/management/api/proxy/properties-ng/api-properties-old.component.ts:56]: I think it's not very important here. At the end, this component will be deleted once the migration is done.

File: gravitee-apim-console-webui/src/management/api/proxy/properties/api-properties.component.ts [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+@Component({
+  selector: 'api-properties',
+  template: require('./api-properties.component.html'),
+  styles: [require('./api-properties.component.scss')],
+})
+export class ApiPropertiesComponent implements OnInit, OnDestroy {
+  private unsubscribe$ = new Subject<void>();
+
+  public isReadOnly = false;
```
Comment (phiz71) [inline gravitee-apim-console-webui/src/management/api/proxy/properties/api-properties.component.ts:47]: Don't forget to compute this value according to origin (Management vs. Kubernetes) and user permissions.

**Review by phiz71 — APPROVED** [review]
LGTM

File: gravitee-apim-console-webui/src/management/api/proxy/properties/api-properties.component.ts (hunk 2 above)
Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/management/api/proxy/properties/api-properties.component.ts:47]: of course, that why i keep it here ^^

**Pull request merged**